Dockerfile

```diff
@@ -1,17 +1,34 @@
-FROM --platform=linux/amd64 registry.fedoraproject.org/fedora-minimal:40
+FROM nixos/nix:2.22.0 AS build-stage
```

Comment:
Without passing `--platform` here (or when calling docker build) I'm pretty sure this will build a darwin executable. (Unless orbstack handles that differently).

This caused us issues in the past where we ended up deploying Darwin executables to prod.

Reply:
See comment below -- it's set up for multiarch now in the makefile